src/components/ResourceOverview/HospitalOverviewPage.tsx

```diff
@@ -101,6 +114,8 @@ const useLayout = makeStyles({
 type LocationState = { updatedResourceId: string | null };
 
 const HospitalOverviewPage: React.FC = () => {
+  const [order, setOrder] = React.useState<Order>('asc');
```

Comment:
minor nit: it's probably marginally better to use a boolean to avoid string comparisons, but it's really not that big a deal
e.g. 
```suggestion
  const [isAscending, setIsAscending] = React.useState<boolean>(true);
```